packages/hooks/src/useIsMt5LoginListStatusPresent.ts

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useMT5AccountsList } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+
+// TODO: Remove this todo once 'open_order_position_status' flag is available in BE response and in type TSocketResponseData<"mt5_login_list">
+type DetailsOfEachMT5LoginId = ReturnType<typeof useStore>['client']['mt5_login_list'][number];
+type TStatus = keyof DetailsOfEachMT5LoginId;
+
+/**
+ * A custom hook to check if the given status flag is present in the mt5_login_list of given account login id
+ */
+const useIsMt5LoginListStatusPresent = (status: TStatus, account_login_id: string): boolean => {
```

Comment:
Why do we need a boolean here at the function?

Reply:
I wrote that while starting to write the function definition, just to tell myself this function returns boolean. removed it now. thanks 👌🏼